utilities/bash/run_six.sh

```diff
@@ -1077,26 +1111,45 @@ function condor_sub(){
 	    done < ${sixdeskjobs}/${LHCDesName}.list
 	else
 	    sixdeskmess -1 "Submission was successful"
-	    # parse terse output (example: "23548.0 - 23548.4")
-	    clusterID=`echo "${terseString}" | head -1 | cut -d\- -f2 | cut -d\. -f1`
-	    clusterID=${clusterID//\ /}
-	    jobIDmax=`echo "${terseString}" | head -1 | cut -d\- -f2 | cut -d\. -f2`
-	    let jobIDmax+=1
-	    nCases=`wc -l ${sixdeskjobs}/${LHCDesName}.list | awk '{print ($1)}'`
-	    if [ ${jobIDmax} -ne ${nCases} ] ; then
-		sixdeskmess -1 "Something wrong with htcondor submission: I requested ${nCases} to be submitted, and only ${jobIDmax} actually made it!"
+	    if [ "$sixdeskplatform" == "htcondor" ]; then
```

Comment:
I think that here HTCondor and HTBoinc behave almost identically, apart from the removal of the .zip for HTBoinc, right? Then I would not split the code with the conditional statement, apart from the loop to rm the .zip files. I am happy if we keep your way of detecting the cluster :)

Reply:
If I'm not mistaken, the difference is also that in HTCondor we use the database, while we don't do so in HTBoinc. E.g. the content of the while loops is different between the two. Personally I would keep it like this for the sake of clarity (maybe move the detection of the cluster in front of the split statement). What do you think?